docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
   nextcloud:
     build:
       context: ./docker
-      dockerfile: Dockerfile.php72
+      dockerfile: Dockerfile.php73
```

Comment:
I'd actually like to keep the default at the minimum supported version of the current Nextcloud master. Could you remove the bumps in here? Otherwise the 7.3 dockerfile looks good :+1:

Reply:
Done!